### PR TITLE
Update default.php in mod_banners

### PR DESCRIPTION
**Pull request by DelPoint**
Missing container element for _$headerText_ (optional in banner module), which limits the header text's CSS styling. (While _$footerText_ has a proper container element.)

Pull Request for Issue # .

### Summary of Changes
Added **div** container element around _$headerText_ (optional **Header Text** in banner module).

### Testing Instructions
Nothing serious, just select 'HTML5' or 'none' style for module output, then check the output code of the page.

### Actual result BEFORE applying this Pull Request
```
<div class="bannergroup">
  Banner Header Text
 <div class="banneritem">
 ...
```
### Expected result AFTER applying this Pull Request
```
<div class="bannergroup">
  <div class="bannerheader">
    Banner Header Text
  </div>
 <div class="banneritem">
 ...
```
### Documentation Changes Required
Add **bannerheader** class name and its description to this document (or to its newer variant, if exist - couldn't find one though): https://docs.joomla.org/CSS_Class_Names_Frontend_in_Version_1.6

### CODE REVIEW

File: modules/mod_banners/tmpl/default.php [hunk 1]
```diff
@@ -13,7 +13,9 @@
 ?>
 <div class="bannergroup<?php echo $moduleclass_sfx; ?>">
 <?php if ($headerText) : ?>
-	<?php echo $headerText; ?>
+	<div class"bannerheader">
```
Comment (Quy) [inline modules/mod_banners/tmpl/default.php]:
```suggestion
	<div class="bannerheader">
```

Comment (DelPoint) [reply modules/mod_banners/tmpl/default.php]: Correct. Although I have no idea how that equal character has missed the pull request :D
Ciao.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 4789d137bc735efbc803bb8a3aab32228f9cd9ac<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32622">issues.joomla.org/tracker/joomla-cms/32622</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 4789d137bc735efbc803bb8a3aab32228f9cd9ac<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32622">issues.joomla.org/tracker/joomla-cms/32622</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32622">issues.joomla.org/tracker/joomla-cms/32622</a>.</sub>

**Comment by HLeithner** [comment]
Rebase on 3.10

**Comment by zero-24** [comment]
Merging thanks 👍

**Pull request merged**